desktop/components/auth_modal/templates/signup.jade

```diff
@@ -11,8 +11,9 @@
 
     section#auth-body
       nav.auth-options
+        - var intentQuery = signupIntent ? '&signup-intent=' + signupIntent : ''
         a.auth-signup-facebook.avant-garde-button-black(
-          href='/users/auth/facebook?redirect-to=' + redirectTo
+          href='/users/auth/facebook?redirect-to=#{redirectTo}' + intentQuery
```

Comment:
I'm not 100% sure if this works, single quotes with interpolation `#{..}`, we should doublecheck.

Reply:
I think I checked it, but probably makes sense to just use `+`s rather than mixing interpolation anyways